scripts/hcnmgr

```diff
@@ -71,9 +69,9 @@ usage() {
 
 # Display current version of hybrid network support
 show_version() {
-	echo "$HCNMGR: Version $VERSION"
+	echo "$HCNMGR (vnic support)"
```

Comment:
Should we say " vnic support " or " IBM veth and IBM vNIC support"?  this will be more clear  I think.

Reply:
This is to be able to tell if the hcnmgr supports vnic from the logs. veth was always supported, it is not a new feature so no need to mention it.

Reply:
but maybe we could instead add something like

features: vnic

so more features can be added to the list later.